pkg/rpc/request/param.go

```diff
@@ -23,6 +23,7 @@ type (
 	// the client.
 	Param struct {
 		json.RawMessage
+		cache interface{}
```

Comment:
I don't think we have a lot of cases with multiple accesses to Param, maybe it's easier to optimize them (to only get the data from Param once) instead of adding this cache?

Reply:
I can optimize them, but the problem is the same as with https://github.com/nspcc-dev/neo-go/pull/2234#discussion_r745487138.

Another problem is that our `GetUint160*`, `GetUint160FromAddressOrHex` and `GetUint256` wrappers can work only with `Param`, so removing `cache` makes us to get corresponding Uint160/Uint256 values manually.